src/pudl/workspace/resource_cache.py

```diff
@@ -8,7 +8,7 @@
 
 import fsspec
 
-from pudl.helpers import fsspec_delete, metafs_exists
+from pudl.helpers import fsspec_delete, fsspec_exists
```

Comment:
this was also potentially a mistake in the prefect branch commit I branched off of.